scripts/exglobal_stage_ic.sh

```diff
@@ -149,7 +171,7 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
 
   # Stage the WW3 initial conditions to ROTDIR (warm start; TODO: these should be placed in $RUN.$gPDY/$gcyc)
   if [[ "${DO_WAVE:-}" = "YES" ]]; then
-    YMD=${PDY} HH=${cyc} declare_from_tmpl COM_WAVE_RESTART
+    YMD=${gPDY} HH=${gcyc} declare_from_tmpl COM_WAVE_RESTART
```

Comment:
If this is now going to use the previous cycle's date, the variable name should be changed to `COM_WAVE_RESTART_PREV` to be consistent.

Also, I believe this will break the existing GEFS test.